.github/workflows/build.yml

```diff
@@ -99,8 +99,7 @@ jobs:
           - path: gabime/spdlog
             ref: v1.6.1
             options: -DSPDLOG_BUILD_EXAMPLE:BOOL=OFF -DSPDLOG_BUILD_SHARED:BOOL=ON
-          - path: humanoid-path-planner/hpp-spline
-            ref: v4.7.0
+          - path: loco-3d/ndcurves
```

Comment:
Since this is not a package under our control I'd rather we fix the version, [v1.1.0](https://github.com/loco-3d/ndcurves/releases/tag/v1.1.0) should be ok